07_safety/01_agency_geo_county.Rmd

```diff
@@ -0,0 +1,731 @@
+
+We do not want to use NIBRS data for some years and UCR data for others. However, because of lack of reporting, **there are a lot of missing values prior to 2021 for NIBRS data**. Future updates of the safety metrics may want to include historical NIBRS data back to 2014 and should keep this in mind.
+
+    Note that we only use the 2022 Batch Header File. Using the Batch Header File from each year might be more accurate, but for the most part it won't matter, especially because so few agencies reported through NIBRS prior to 2021. Additionally, downloading and reading in the Batch Header File for each year takes a lot of time and computing power, and must be done manually for each year which introduces room for error in replication. Future updates may consider using the Batch Header Files for each year, which can be accessed through the following links:
+    * [2014](https://www.icpsr.umich.edu/web/ICPSR/studies/36421)
+    * [2015](https://www.icpsr.umich.edu/web/ICPSR/studies/36851)
+    * [2016](https://www.icpsr.umich.edu/web/ICPSR/studies/37066)
+    * [2017](https://www.icpsr.umich.edu/web/ICPSR/studies/37650)
+    * [2018](https://www.icpsr.umich.edu/web/ICPSR/studies/37649)
+    * [2019](https://www.icpsr.umich.edu/web/ICPSR/studies/38565) 
+    * [2020](https://www.icpsr.umich.edu/web/ICPSR/studies/38566) 
+    * [2021](https://www.icpsr.umich.edu/web/ICPSR/studies/38807)
+    * [2022](https://www.icpsr.umich.edu/web/ICPSR/studies/38925)
+    
+Accessing these data requires an ICPSR account. Manually downloading these data is a five-step process, but you will only need to do each of these steps once for the creation of this metric:
+    1. Create an ICPSR account [here](https://login.icpsr.umich.edu/realms/icpsr/login-actions/registration?client_id=icpsr-web-prod&tab_id=BqotoE6wnqw).
+    2. Navigate to the National Archive of Criminal Justice Data (NACJD) landing page for the year you are accessing (e.g., 2022 is [here](https://www.icpsr.umich.edu/web/NACJD/studies/38925)).
+    3. Select `Download`
+        + Select `Delimited`
+        + Select `Agree` on the `Terms of Use`
+    4. Sign into your ICPSR account
+    5. This will download a zip file titled `ICPSR_38925-V2`. Unzip the file, navigate to `ICPSR_38925\DS0001\38925-0001-Data.tsv`, and move it to the appropriate directory/folder. This is the `Batch Header File`.
```

Comment:
I would clarify that the appropriate directory/folder you refer to in this script is the 07_safety/data folder in the github directory

Reply:
I clarified this and added a disclaimer at the beginning of the file that the code assumes the user has cloned the repo from GitHub with instructions on how to do so.